virttest/virt_vm.py

```diff
@@ -1110,62 +1052,65 @@ def print_guest_network_info():
             """
             Print guest network information into debug log file
             """
-            session = self.serial_login(internal_timeout, username, password, False)
+            session = None
```

Comment:
You don't need to assign `session` to `None`, as self.serial_login will return ShellSession object on success and None on failure

Reply:
Should have this declaration, or latter in `finally` section, it will get `NameError: name 'session' is not defined`.